Cargo.toml

```diff
@@ -198,7 +198,7 @@ members = [
 
 [profile.dev]
 opt-level = 0
-debug = false
+debug = 1
```

Comment:
Is it required?

Reply:
no, just to see backtrace with line number temporarily